src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div class="container"
+      data-qa = "card">
+      <img src="./images/imac.jpeg"
+        alt="imac"
+        class="container__image">
+      <div class="container__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="container__kod">
+        Код товара: 195434
+      </div>
+      <div class="container__stars">
+        <div class="stars">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
Тут еще должен быть класс-модификатор, переиспользуй код из задания layout_stars